Proofing your index changes (#44)
yayoi-chi/yaya-world: content/en/introduction.org

Hewwo hewwo nyaoo nyaoo~~

> My name is Yayoi Shizumi (静海やよい in Japanese) and I am a magical fairy cat princess from a mystical realm near the North West Passage! I was sent to this realm to protect it. Unfortunately, my travel here was far from uneventful and I am no longer able to speak. I have had to find new ways to communicate in order to protect those I was sent +to protect+ (here for, "protect" seems redundant here), my kittens!

Reviewer: I think you're right, "protect" does seem redundant. "I was sent here for" doesn't quite seem like something i'd say though. How about "I have had to find new ways to communicate in order to fulfill my mission (or task)"?

Author: I like "I have had to find new ways to communicate in order to fulfill my mission", but you would lose the reference about your kittens, so maybe something like:

```suggestion
My name is Yayoi Shizumi (静海やよい in Japanese) and I am a magical fairy cat princess from a mystical realm near the North West Passage! I was sent to this realm to protect it. Unfortunately, my travel here was far from uneventful and I am no longer able to speak. I have had to find new ways to communicate in order to fulfill my mission, and protect my kittens!
```

You can get recaps, and updates on this website, which the kind folks taking care of me in your world have made for me, but you can also get updates on [[https://mastodon.world/@yaya][Mastodon]] and my [[https://www.youtube.com/@yayoi-chi/community][YouTube Community Tab]]! I love helping and have also created some [[{{% ref guides %}}][Guides]] and [[{{% ref reviews %}}][Reviews]] of some of my favorite games for you!

My mission above all is to be a great friend! So please help me out and connect with me. I might not be able to talk anymore, but I can still communicate and I love getting to know new people. Also, if you would like to become one of my kittens, I'd love to have you. Just follow me on Twitch or subscribe on YouTube!

> Thank you so so so much for +listening+ (reading?)! I hope I will be able to get to know you!

Reviewer: You commit this one has is. I should probably not use words that refer to spoken language 😅
